### ISSUE TITLE
A "raw" link to another notion link is dropped

### ISSUE THREAD

**Issue by hatton** [souvikinator/notion-to-md#31]
A raw link to another website, e.g. `This is a bare to google https://www.google.com/` is emitted as 
`This is a bare to google [https://www.google.com/](https://www.google.com/)`

A bit of text with an underlying link to another page 
![image](https://user-images.githubusercontent.com/8448/169856265-7b69439f-40e6-4249-898f-f6021d82384a.png)

is emitted as 
`This is an inline link another page [Apples](/c62dbc3fede94264a1d5e0245fa4be73).`

However, if you have a link *without* a text label, then notion-to-md just drops it. This
![image](https://user-images.githubusercontent.com/8448/169856110-2a187a05-4ba9-49ae-ba48-3efab88ad087.png)
is emitted as
`This is a raw link to the intro: `

In the content returned by the API, this last case comes as this kind of block

```javascript
 {
      "object": "block",
      "id": "5f4fdb6f-8adb-4cff-9137-2168530b78bf",
      <snip>
      "type": "link_to_page",
      "link_to_page": {
        "type": "page_id",
        "page_id": "8b16a919-6add-4e37-b1a4-4103feb24d5d"
      }
}
```

**Comment by souvikinator** [comment]
Hey there, 
Thanks for reporting the issue. I'll push a new release once the bug is fixed.

**Comment by hatton** [comment]
I appreciate you looking into this, but I'm baffled.

The API has supported linked pages since November 2021. See [Synced Block, Link to Page and Template block types are now supported in the API](https://developers.notion.com/changelog/synced-block-link-to-page-and-template-block-types-are-now-supported-in-the-api). @notionhq/client has supported it since 0.4.8. https://github.com/makenotion/notion-sdk-js/releases/tag/v0.4.8.

**Comment by souvikinator** [comment]
May I know the version of @notionhq/client you are using? 
This side it's v1.0.4 (latest). 
I tired it on different notion pages and databases as well but it still shows `type: unsupported`.

Mind sharing the code snippet you are using to get the response object?

**Comment by hatton** [comment]
Sure, here are full steps:

1) Duplicate this page: https://hattonjohn.notion.site/link_page-sample-a73ae52733ae4e98bc46923f0e7459b6
![image](https://user-images.githubusercontent.com/8448/170717097-59a6b50b-bbbc-4a01-a03e-4f4891906d2b.png)

2) Share your own integration with your page

3) Code like this:

```
 "@notionhq/client": "^1.0.4",
```

```
import { Client } from "@notionhq/client";
import { env } from "process";

const notionClient = new Client({
  auth: env.YOUR_INTEGRATION_TOKEN,
});

notionClient.blocks.children
  .list({
    block_id: "<YOUR PAGE ID>",
  })
  .then(b => console.log(JSON.stringify(b, null, 2)))
  .catch(e => console.error(e));
```

```
$ ts-node src/link_page.ts
{
 ...
    {
      "object": "block",
      "id": "426ae49a-7373-47a8-992c-c5df7a9bf588",
      "created_time": "2022-05-27T14:00:00.000Z",
      "last_edited_time": "2022-05-27T14:00:00.000Z",
      "created_by": {
        "object": "user",
        "id": "11fb7f16-0560-4aee-ab88-ed75a850cfc4"
      },
      "last_edited_by": {
        "object": "user",
        "id": "11fb7f16-0560-4aee-ab88-ed75a850cfc4"
      },
      "has_children": false,
      "archived": false,
      "type": "link_to_page",
      "link_to_page": {                      <-------------------
        "type": "page_id",
        "page_id": "5d4f91d1-80cd-4a36-a284-905905c92001"
      }
    }
}
``

**Comment by hatton** [comment]
Submitted PR #32

**Comment by souvikinator** [comment]
hey there @hatton 
Thanks for the snippet, I was using the same snippet, got following results:

<img width="316" alt="image" src="https://user-images.githubusercontent.com/64456160/171899326-0252332c-bfe8-4ac7-a4b0-a49924542700.png">

so not sure what's the issue over here and the same goes for the changes in PR #32
Is the PR working from your side?

**Comment by hatton** [comment]
> Is the PR working for you?

Yes, of course. It seems like at this point maybe you could file a bug report with Notion to try and figure out what is going on with your account. For me, everything is working as Notion documents it to work.

**Comment by souvikinator** [comment]
Yes, I did that already and waiting for their response on slack. Once it is resolved I'll merge and create a release.

**Comment by souvikinator** [comment]
@hatton 
Seems like no one is responding but if the PR works for you I'll merge it and create a new release.
